### PR TITLE
Adding a Blackhole to the space dimension

### PR DESCRIPTION
**Pull request by almajd3713**
I kind of wished there was a blackhole background for my space base (well, future), so I just added that for the planetDefs file. 
You can change the settings for it to make it more distant and consume more fuel for example, or just to change its name.

### CODE REVIEW

File: overrides/config/advRocketry/planetDefs.xml [hunk 1]
```diff
@@ -112,4 +112,5 @@
 			<rotationalPeriod>21100</rotationalPeriod>
         </planet>
 		</star>
-</galaxy>
\ No newline at end of file
+    <star name="Eye Of Eternity" numGasGiants="0" numPlanets="0" size="2.0" temp="800" x="300" y="120" blackHole="true" > </star>
```
Comment (IntegerLimit) [inline overrides/config/advRocketry/planetDefs.xml:115]: This should be a tab indentation, to fit with the rest of the document.

Comment (almajd3713) [reply overrides/config/advRocketry/planetDefs.xml:115]: My bad, my editor uses spaces instead of tabs which I will change now. Thanks for the heads up!

**Comment by almajd3713** [comment]
Okay uh as it turned out, Sol does indeed have a blackhole background for some reason. Although in my initial testing I am 120% sure there wasn't anything but a void there. Closing this pr. Thanks for your time.

**Comment by IntegerLimit** [comment]
Does it? Where?

**Pull request closed without merging**